modules/framework/galasa-parent/dev.galasa.framework.api.cps/src/main/java/dev/galasa/framework/api/cps/internal/routes/AddPropertyInNamespaceRoute.java

```diff
@@ -37,17 +40,20 @@ public class AddPropertyInNamespaceRoute extends CPSRoute {
     private String propertyName;
     private String namespaceName;
     
-    public AddPropertyInNamespaceRoute(ResponseBuilder responseBuilder, IFramework framework) {
+    public AddPropertyInNamespaceRoute(ResponseBuilder responseBuilder, IFramework framework, Environment env) throws RBACException {
```

Comment:
Can we wrap the request, to be a requestContext, which contains the loginId of the caller + the original request ?
Then we don't need to pass the env down everywhere.